packages/components/src/index.js

```diff
@@ -137,7 +137,7 @@ export { Truncate as __experimentalTruncate } from './truncate';
 export {
 	default as __experimentalUnitControl,
 	useCustomUnits as __experimentalUseCustomUnits,
-	ALL_CSS_UNITS,
+	parseUnit,
```

Comment:
maybe make it experimental like the two others as they work together?

Reply:
Done :+1: